engine/pom.xml

```diff
@@ -661,6 +658,28 @@
       </build>
     </profile>
 
+    <profile>
+      <id>cockroachdb</id>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <failIfNoTests>false</failIfNoTests>
+              <trimStackTrace>false</trimStackTrace>
+              <redirectTestOutputToFile>true</redirectTestOutputToFile>
+              <excludes combine.children="append">
+                <exclude>**/CompetingMessageCorrelationTest.java</exclude>
+                <exclude>**/DbDeadlockTest.java</exclude>
+                <exclude>**/CompetingHistoricVariableInstancePartitioningTest.java</exclude>
+                <exclude>**/CompetingHistoricByteArrayPartitioningTest.java</exclude>
```

Comment:
Some of these tests have CockroachDB-specific code now. This is not needed if we ignore them anyway.

Reply:
I have removed the whole `excludes` block as all of these tests have code that either handles CRDB scenarios or have annotations that include/exclude the appropriate tests on CRDB.